spec/controllers/activity_forms_controller_spec.rb

```diff
@@ -39,6 +39,18 @@
         end
       end
 
+      context "ispf_theme step" do
+        subject { get_step :ispf_theme }
+
+        it { is_expected.to skip_to_next_step }
+
+        context "when it's an ISPF activity" do
+          let(:activity) { create(:project_activity, organisation: organisation, parent: fund, source_fund_code: Fund.by_short_name("ISPF").id) }
```

Comment:
Given that the project side of this is coming in a later ticket, should this be a programme activity here? It seems like project might be correct in this context though

Reply:
Good point, this should be a `programme` which means the previous tests were incorrect too. I've updated them in their own commit.